### ISSUE TITLE
Parsing step sizes and other parameters

### ISSUE THREAD

**Issue by langner** [cclib/cclib#260]
Parsing the step size (nuclear, electronic, magnetic) came up in #225. Would be nice to have, and generally we should think about how to support settings like this which are not outputs of the computation.

**Comment by langner** [comment]
This is something that can go into `metadata` now.

**Comment by langner** [comment]
Pushing back. @berquist do you think this belongs in `metadata` the way things have been going?

**Comment by berquist** [comment]
Yes, it looks like we have most things that would go in an input parser inside of `metadata`.
